CHANGELOG.md

```diff
@@ -50,8 +50,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 
 ### Removed
+- AnyToMetadataClass and AnyToMetadata type. Neither was really used anywhere
 
 ### Fixed
+- CIP-25 strings are now being split into chunks whose sizes are less than or equal to 64 to adhere to the CIP-25 standard ([#1343](https://github.com/Plutonomicon/cardano-transaction-lib/issues/1343))
```

Comment:
This isn't a general change for CIP-25 strings is it? It's just for the `image` field?

Reply:
According to the standard (where necessary). I think this description is good.

Reply:
And the `description` field it looks like

Reply:
Right makes sense